package.json

```diff
@@ -8,20 +8,24 @@
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
     "build": "tsc && vite build || echo 'Build failed'",
-    "build:watch": "tsc && vite build --watch"
+    "build:watch": "tsc && vite build --watch",
+    "test": "jest"
   },
   "devDependencies": {
+    "@types/jest": "^29.5.12",
+    "@types/uuid": "^9.0.8",
     "@typescript-eslint/eslint-plugin": "^7.0.2",
     "@typescript-eslint/parser": "^7.0.2",
     "eslint": "^8.56.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.1.3",
+    "jest": "^29.7.0",
     "prettier": "^3.2.5",
-    "@types/uuid": "^9.0.8",
-    "typescript": "^5.2.2",
+    "ts-jest": "^29.1.2",
+    "typescript": "^5.4.2",
```

Comment:
Why update the typescript version?
Because of this update. I am getting this warning when i run "npm run lint"
![image](https://github.com/lightsheet-team/lightsheet/assets/24876640/77bf2148-0838-4d3d-8eb4-638109266a7d)

Reply:
Left the old version!